File: test/bench/b1.lua
-- Lua transcription of b1 (empty loop, 100 M iterations).
-- Run: lua5.4 b1.lua
-- Matches the Forth `1 drop` body with a trivial local store; Lua's
-- `for ... do end` would otherwise be a single VM op with no body.
for i = 1, 100000000 do local x = 1 end

File: test/bench/b1.py
# Python 3 transcription of b1 (empty loop, 100 M iterations).
# Run: python3 b1.py
# Matches the Forth `1 drop` body with a trivial local store so the
# loop is not optimised to a no-op.
for i in range(100000000):
    x = 1
